packages/web3.js-plugin-fee-estimation/.prettierrc.cjs

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ...require('../../.prettierrc.js'),
```

Comment:
Unrelated to PR. We should move the prettierrc to a package sometime. It's a tiny antipattern to require files outside of a package without depending on them. I haven't gotten around to doing this yet though because it's super super low priority though